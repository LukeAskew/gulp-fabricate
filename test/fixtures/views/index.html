---
title: Document Name
name: World
form-desc: Lorem ipsum dolor sit amet, consectetur.
---

<h1>{{home.greeting}}, {{name}}!</h1>

<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Veritatis numquam eaque ea enim tempore, quaerat, reprehenderit, ut eos incidunt impedit maiores quibusdam porro? Saepe, debitis tenetur voluptates necessitatibus, ipsa nesciunt.</p>

{{> button}}

{{> form form-desc="A cool form."}}

{{#each materials}}

	<h1>{{this.name}}</h1>

	{{#each this.items}}

		<h2>{{this.name}}</h2>

		{{#if this.notes}}<div class="notes">
			{{{this.notes}}}
		</div>{{/if}}

		<div class="item">
			{{partial @key @root}}
		</div>

	{{/each}}

{{/each}}
