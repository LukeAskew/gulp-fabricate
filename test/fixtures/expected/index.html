<!doctype html>
<html lang="en">
<head>

    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">

    <title>Document Name</title>

    <meta name="viewport" content="width=device-width, initial-scale=1">

    <link rel="stylesheet" href="assets/styles/main.css">

</head>
<body>

    <h1>Hello, World!</h1>

	<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Veritatis numquam eaque ea enim tempore, quaerat, reprehenderit, ut eos incidunt impedit maiores quibusdam porro? Saepe, debitis tenetur voluptates necessitatibus, ipsa nesciunt.</p>


	<a href="#" class="button">Click Me, World</a>


	<form action="">
		<p>A cool form.</p>
		<input type="text" placeholder="Luke">
		<a href="#" class="button">Click Me, Universe</a>
	</form>


	<h1>Components</h1>

	<h2>Button</h2>

	<div class="notes">
		<p>foo <code>bar</code></p>
	</div>

	<div class="item">
		<a href="#" class="button">Click Me, World</a>
	</div>

	<h1>Structures</h1>

	<h2>Form</h2>

	<div class="item">
		<form action="">
			<p>Lorem ipsum dolor sit amet, consectetur.</p>
			<input type="text" placeholder="Luke">
			<a href="#" class="button">Click Me, Universe</a>
		</form>
	</div>

    <script src="assets/scripts/main.js"></script>

</body>
</html>
